docs/reference/cat/shards.asciidoc

```diff
@@ -78,7 +78,9 @@ twitter 0 p RELOCATING 3014 31.1mb 192.168.56.10 H5dfFeA -> -> 192.168.56.30 bGG
 [[states]]
 ==== Shard states
 
-Before a shard can be used, it goes through an `INITIALIZING` state.
+Before a shard can be used, it goes through an `INITIALIZING` or `Relocating` state.
```

Comment:
I don't quite understand this change, and think the original docs are more correct than the statement written here. All shards go through an `INITIALIZING` state before they can be used, and moreover a shard that is in the `RELOCATING` state is completely usable - `RELOCATING` is much more like `STARTED` than `INITIALIZING`.